cigaradvisor/blocks/article-teaser/article-teaser.css

```diff
@@ -142,9 +142,15 @@
   display: block;
 }
 
-@media screen and (min-width: 60em) {
+@media screen and (min-width: 1200px) {
   .article-teaser.block .article.article-thumbnail .article-header {
-    min-height: 140px
+    min-height: 140px;
+  }
+}
+
+@media screen and (min-width: 600px) and (max-width: 900px) {
```

Comment:
For this. make the default be min-with 600px or smaller. Then add a media query for min-width 900px, then another at 120px.

That way it's "Default is smallest screen"
Override at larger
Override again at larger.

Keeps thing succinct. And keep all queries at the end, in ascending min-width order

Reply:
Refactored as suggested.